kalamine/layout.py

```diff
@@ -287,13 +309,13 @@ def _parse_template(self, template: str, rows: List[str], layer_number: Layer):
                         # shift_key = base_key.upper()
 
                 if base_key != " ":
-                    self.layers[layer_number + 0][key] = base_key
+                    self.layers[layer_number][key] = base_key
                 if shift_key != " ":
-                    self.layers[layer_number + 1][key] = shift_key
+                    self.layers[layer_number.next()][key] = shift_key
```

Comment:
This looks like a very valid reason to kill my beautiful vertical alignment. Thanks for the expressiveness.